.env.example

```diff
@@ -56,3 +56,5 @@ JEDI_TOKEN=
 
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
+
+TEAM_SLACK_WEBHOOK_URL=
```

Comment:
Are you sure you can use a single Slack webhook for more than one channel?

Reply:
Yes, the screenshot was made using the #onken-alerts hook but with the destination set to @joldenburg, so it came as a DM to me.